docs/architecture/porting-existing-aspnet-apps/configuration-differences.md

```diff
@@ -7,8 +7,101 @@ ms.date: 11/13/2020
 
 # Configuration differences between ASP.NET MVC and ASP.NET Core
 
+How configuration values are stored and read changed dramatically between ASP.NET and ASP.NET Core.
+
+## ASP.NET MVC configuration
+
+In ASP.NET apps, configuration uses the built-in .NET configuration files, web.config in the app folder and machine.config on the server. Most ASP.NET MVC and Web API apps store any settings they need in the configuration file's appsettings or connection strings elements. Some also use custom configuration sections that can be mapped to a settings class.
+
+Configuration in .NET applications is accessed using the `System.Configuration.ConfigurationManager` class. Unfortunately, this class provides static access to the config elements, and as a result very few ASP.NET MVC apps tend to abstract access to their configuration settings or inject them where needed. Instead, most .NET apps tend to directly access the configuration system anywhere the app needs to use a setting.
+
+Typical ASP.NET MVC configuration access:
+
+```csharp
+string connString = ConfigurationManager.ConnectionStrings["DefaultConnection"]
+    .ConnectionString;
+```
+
+## ASP.NET Core configuration
+
+In ASP.NET Core apps, configuration is, itself, configurable. However, most apps use a set of defaults provided as part of the standard project templates and the `ConfigureWebHostDefaults` method used in them. The default settings use JSON formatted files, with the ability to override settings in the base `appsettings.json` file with environment-specific files like `appsettings.Development.json`. Additionally, the default configuration system further overrides all file-based settings with any environment variable that exists for the same named setting. This is useful in a number of scenarios, and is especially useful when deploying to a hosting environment, since it eliminates the need to worry about whether deploying configuration files will accidentally overwrite important production configuration settings.
```

Comment:
```suggestion
In ASP.NET Core apps, configuration is, itself, configurable. However, most apps use a set of defaults provided as part of the standard project templates and the `ConfigureWebHostDefaults` method used in them. The default settings use JSON formatted files, with the ability to override settings in the base `appsettings.json` file with environment-specific files like `appsettings.Development.json`. Additionally, the default configuration system further overrides all file-based settings with any environment variable that exists for the same named setting. This is useful in a number of scenarios and is especially useful when deploying to a hosting environment, since it eliminates the need to worry about whether deploying configuration files will accidentally overwrite important production configuration settings.
```